compiler/damlc/tests/src/DA/Test/GenerateSimpleDalf.hs

```diff
@@ -0,0 +1,83 @@
+-- Copyright (c) 2019 The DAML Authors. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+module DA.Test.GenerateSimpleDalf (main) where
+
+import qualified Data.ByteString.Lazy as BSL
+import qualified Data.NameMap as NM
+import qualified Data.Text.IO as T
+import System.Environment
+
+import DA.Daml.LF.Ast.Base
+import DA.Daml.LF.Ast.Util
+import DA.Daml.LF.Ast.Version
+import DA.Daml.LF.Ast.World
+import DA.Daml.LF.Proto3.Archive
+import DA.Daml.LF.TypeChecker
+import DA.Pretty
+
+-- | This tool generates a simple DALF file and writes it to the first
+-- argument given on the command line. This DALF is intended to be used
+-- as a test case for the plain DALF import feature.
+main :: IO ()
+main = do
+    [file] <- getArgs
+    let version = V1 (PointStable 6)
+    let modName = ModuleName ["Module"]
+    let modRef = Qualified PRSelf modName
+    let tplFields = map FieldName ["this", "arg"]
+    let tplRec = DefDataType
+            { dataLocation = Nothing
+            , dataTypeCon = TypeConName ["Template"]
```

Comment:
:see_no_evil:

Reply:
Let's make sure we import the stdlib qualified before our users find out we don't do it. ;)